### DISCUSSION TITLE
want Galaxy Brain badges

### DISCUSSION THREAD

**Discussion by vitorRibeiro7** [Schweinepriester/github-profile-achievements#421] [Q&A]
Does the achievement mean:
"You need to open 2 discussions that are answered"
or
"You need to become the marked answer to 2 discussions?"

Or it's another different definition?

**Comment by Schweinepriester** [comment]
It's "You need to become the marked answer to 2 discussions"

**Reply by vitorRibeiro7** [reply]
idk how do this :v

**Comment by etmjr03** [comment]
Will it be necessary to have two answers accepted in two discussions?
